tcms/kiwi_auth/tests/test_backends.py

```diff
@@ -0,0 +1,20 @@
+from django.test import TestCase
```

Comment:
Since you are organizing the tests inside a module `git mv` the rest of the test files from the kiwi_auth directory here. There's also a `tests.py` file which has the same name as the module and this messes up the test runner b/c you can't have 2 modules with the same name in the same directory.